ecommerce/api.py

```diff
@@ -117,7 +108,7 @@ def generate_cybersource_sa_payload(order, base_url):
         "reference_number": make_reference_id(order),
         "profile_id": settings.CYBERSOURCE_PROFILE_ID,
         "signed_date_time": now_in_utc().strftime(ISO_8601_FORMAT),
-        **overrides,
+        "override_custom_receipt_page": urljoin(base_url, "dashboard/"),
```

Comment:
Should `override_custom_cancel_page` still be included too?

Reply:
No, we will be using whatever the CyberSource default is for that